.envrc

```diff
@@ -1 +1,2 @@
+nix_direnv_watch_file ./pre-commit.nix
```

Comment:
Was is das? Just curious :)

Reply:
I think I mentioned this in our session, we can set direnv to reload automatically, when some files are changed (this takes multiple arguments)